app/src/main/kotlin/com/hippo/ehviewer/spider/DownloadInfoMagics.kt

```diff
@@ -0,0 +1,29 @@
+package com.hippo.ehviewer.spider
+
+import com.hippo.ehviewer.client.data.GalleryInfo
+import com.hippo.ehviewer.client.thumbUrl
+import com.hippo.ehviewer.dao.DownloadInfo
+
+object DownloadInfoMagics {
+    private const val DOWNLOAD_INFO_DIRNAME_URL_MAGIC = "Lolicon"
+    private const val DOWNLOAD_INFO_DIRNAME_URL_SEPARATOR = "~"
```

Comment:
Don't use this as dirname may contain it.

Reply:
I'd say use `|`, which is not allowed in both URLs and filenames.

Reply:
SGTM